draft-ietf-httpbis-rand-access-live.xml

```diff
@@ -439,6 +439,9 @@ Content-Range: bytes 1020000-999999999999/*
       <t>
         with the response body returning bytes 1020000-1254567 immediately and aggregated ("live") data being returned as the content is aggregated.
       </t>
+      <t>
+        While it would be ideal to define a single numerical Very Large Value, there's no single value that would work for all applications and platforms. e.g. JavaScript numbers cannot represent integer values above 2^^53, so a JavaScript application may want to use 2^^53-1 for a Very Large Value. This value, however, would not be sufficient for all applications. So this draft recommends 2^^53-1 (9007199254740991) as a Very Large Value unless an application has a good justification to use a smaller or larger value. e.g. If it's always known that the resource won't exceed a value smaller than the recommended Very Large Value for an application, a smaller value can be used for efficiency. And if it's likely that an application will utilize resources larger than the recommended Very Large Value, a larger value should be used.
```

Comment:
"e.g." is followed by a comma: "e.g.,"

And I think that you can drop "for efficiency" on the second to last sentence.

Reply:
Agreed. Thanks as always Martin!